Add package libcmpiutil version 0.5.7

suresh-thelkar opened the pull request:
<!--
COMMENT BLOCKS WILL NOT BE INCLUDED IN THE PR.
Feel free to delete sections of the template which do not apply to your PR, or add additional details
-->

###### Merge Checklist  <!-- REQUIRED -->
<!-- You can set them now ([x]) or set them later using the Github UI -->
**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary <!-- REQUIRED -->
<!-- Quick explanation of the changes. -->
What does the PR accomplish, why was it needed?
Add package libcmpiutil version 0.5.7

###### Change Log  <!-- REQUIRED -->
<!-- Detail the changes made here. -->
<!-- Please list any packages which will be affected by this change, if applicable. -->
<!-- Please list any CVES fixed by this change, if applicable. -->
- NA

###### Does this affect the toolchain?  <!-- REQUIRED -->
<!-- Any packages which are included in the toolchain should be carefully considered. Make sure the toolchain builds with these changes if so. -->
<!-- Update: manifests/package/toolchain_*.txt, pkggen_core_*.txt, update_manifests.sh -->
<!-- To validate: make clean; make workplan REBUILD_TOOLCHAIN=y DISABLE_UPSTREAM_REPOS=y CONFIG_FILE="" ... -->
**NO**

###### Associated issues  <!-- optional -->
<!-- Link to Github issues if possible. -->
<!-- you can use "fixes #xxxx" to auto close an associated issue once the PR is merged -->
- NA

###### Links to CVEs  <!-- optional -->
- NA

###### Test Methodology
<!-- How was this test validated? i.e. local build, pipeline build etc. -->
Buddy Build
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=293418&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=293419&view=results)

mfrw commented on SPECS-EXTENDED/libcmpiutil/libcmpiutil.spec:
```diff
@@ -0,0 +1,180 @@
+Name:           libcmpiutil
+Version:        0.5.7
+Release:        20%{?dist}
+License:        LGPLv2+
```
```suggestion
License:        LGPL-2.0-or-later
```

mfrw commented on SPECS-EXTENDED/libcmpiutil/libcmpiutil.spec:
```diff
@@ -0,0 +1,180 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libvirt.org/CIM/
+Source:         ftp://libvirt.org/libvirt-cim/libcmpiutil-%{version}.tar.gz
```
Would it be possible to have a `https` source ?

suresh-thelkar replied:
I am now using https instead of ftp. Please look at the latest commit.

oliviacrain commented on SPECS-EXTENDED/libcmpiutil/libcmpiutil.spec:
```diff
@@ -0,0 +1,180 @@
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libvirt.org/CIM/
```
Would suggest `https://gitlab.com/libvirt/libvirt-cim`

suresh-thelkar replied:
Thanks. I have modified accordingly.  Please take a look.

suresh-thelkar commented:
@mfrw  / @oliviacrain, Thanks for the review. I have addressed your comments. Please look at the latest commits. 
I have also reran Buddy Builds after that. Both of them passed. Here are the results. 
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=304858&view=results)
- [ARM64 ](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=304859&view=results)

suresh-thelkar commented:
Thanks @mfrw / @oliviacrain for the approvals. I have once again rebased with mainline. Ran the Buddy Builds. Both of them looks good. Going to merge the PR shortly. 
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305779&view=results)
- [ARM4](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=305780&view=results)
